.github/workflows/build-pack-release.yml

```diff
@@ -93,13 +96,19 @@ jobs:
             role-to-assume: ${{ secrets.NUGET_ACCESS_ROLE_ARN }}
             aws-region: us-west-2
 
-        # TODO: uncomment this step once the rest of the workflow has been verified in prod.
-        #- name: Push packages to Nuget.org
-        #  run: |
-        #    $nugetKey = aws secretsmanager get-secret-value --secret-id ${{ secrets.NUGET_SECRETS_ID }} --region us-west-2 --output text --query SecretString
-        #    nuget push .\Deployment\nuget-packages\*.nupkg -Source https://api.nuget.org/v3/index.json -ApiKey $nugetKey
+        - name: Push packages to Nuget.org
+          run: >
+            $nugetKey = aws secretsmanager get-secret-value
+            --secret-id ${{ secrets.NUGET_SECRETS_ID }}
+            --region us-west-2
+            --output text
+            --query SecretString
+
+            nuget push
+            .\Deployment\nuget-packages\*.nupkg
+            -Source https://api.nuget.org/v3/index.json
+            -ApiKey $nugetKey
             
-        # TODO: Change this step to directly create a public release once the workflow has been verified.
```

Comment:
@willarmiros I know we talked earlier to make this step create an actual release. But I think we can keep this as draft just to be able to edit the release if needed. For example, the 3.x release will need to be a pre-release, and so on.